src/base/bittorrent/session.cpp

```diff
@@ -4091,4 +4120,35 @@ namespace
             qDebug() << Q_FUNC_INFO << " fails: " << exc.what();
         }
     }
+
+#ifdef Q_OS_WIN
+    QString convertIfaceNameToGuid(const QString &name)
+    {
+        // Under Windows XP or on Qt version <= 5.5 'name' will be a GUID already.
+        QUuid uuid(name);
+        if (!uuid.isNull())
+            return uuid.toString().toUpper(); // Libtorrent expects the GUID in uppercase
+
+        typedef LONG (WINAPI *PCONVERTIFACENAMETOLUID)(const WCHAR *, PNET_LUID);
+        PCONVERTIFACENAMETOLUID ConvertIfaceNameToLuid = (PCONVERTIFACENAMETOLUID)::GetProcAddress(::GetModuleHandleW(L"Iphlpapi.dll"), "ConvertInterfaceNameToLuidW");
```

Comment:
`static_cast<PCONVERTIFACENAMETOLUID>`, please. Or `reinterpret_cast` if static one is impossible.